README.md

```diff
@@ -21,24 +21,23 @@ alignmentUrl   |            |             | schema:targetUrl
 
 ## Actions
 
-Action              | IRI       | Description
---------------------|-----------|------------
-DefinedIssuer       | TBA       | Defined a new Issuer Profile
-DefinedBadge        | TBA       | Defined a new BadgeClass
-IssuedBadge         | TBA       | Awarded an instance of a BadgeClass to a recipient, perhaps accepting an application
-MetCriteria         | TBA       | Recipient has met criteria for a BadgeClass and should be awarded a badge.
-Updated             | TBA       | Updated a previously defined badge object with new metadata
-RevokedBadge        | TBA       | Updated a previously awarded badge to mark it as revoked
-Deleted             | TBA       | Marked a defined badge object as deleted
-Registered          | TBA       | Registered a recipient profile with a backpack endpoint or issuer profile with issuer endpoint
-Requested           | TBA       | Requested a badge object or related resource from an issuer or repository
-Verified            | TBA       | Verified an Open Badge's authenticity and structural integrity
-Shared              | TBA       | Triggered or authorized a share of a single badge, collection, or other document including badges
-Endorsed            | TBA       | Issued an endorsement or annotation of an Issuer, BadgeClass, or Assertion that should be published
-Displayed           | TBA       | Displayed badge details on a single badge to a consumer
-Accepted            | TBA       | Inspected a badge or pathway with completion data and accepted it in exchange for some privilege, opportunity, access, or other benefit granted to the recipient.
-Applied             | TBA       | User submits evidence to an issuer in hopes of earning a badge.
-AssignedToObjective | TBA       | Set BadgeClass as related to prospective opportunity, privilege, objective, learning pathway, skill or competency definition
+Action | IRI | Description | New   |
+-------|-----|------------ | :---: |
+Defined | http://purl.imsglobal.org/vocab/caliper/action#Defined | Defined a new BadgeClass or Issuer | Yes |
+Issued  | http://purl.imsglobal.org/vocab/caliper/action#Issued | Awarded an instance of a BadgeClass to a recipient, perhaps accepting an application | Yes
+MetCriteria | http://purl.imsglobal.org/vocab/caliper/action#MetCriteria | Recipient has met criteria for a BadgeClass and should be awarded a badge. | Yes
```

Comment:
Some alignment with the Learning Objects work here for the concept of "meeting some criteria for some outcome", like 'Demonstrated'?

I _think_ it would probably be good to have a general goal of keeping our verb collection smaller in population rather than larger, especially when it comes to verbs that can get shared across profiles...